server/server.go

```diff
@@ -118,6 +120,19 @@ type ArgoCDServerOpts struct {
 	RepoClientset   reposerver.Clientset
 }
 
+// initializeDefaultProject creates the default project if it does not already exist
+func initializeDefaultProject(opts ArgoCDServerOpts) error {
+	defaultProj := &v1alpha1.AppProject{
+		ObjectMeta: metav1.ObjectMeta{Name: common.DefaultAppProjectName, Namespace: opts.Namespace},
+		Spec:       v1alpha1.AppProjectSpec{},
```

Comment:
I mentioned earlier, but the default project that we want to initialize, will be the one with `*` to repos, and `*`/`*` to clusters/namespaces. 

NOTE that we now need to teach the meaning of `*` in the authorization logic.